src/current/_includes/cockroachcloud/free-cluster-limit.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-Organizations without billing information on file can only create one CockroachDB {{ site.data.products.serverless }} cluster.
+Organizations without billing information on file can only create one CockroachDB {{ site.data.products.basic }} cluster.
```

Comment:
Should we say "Organizations without a payment method configured can create only one...."? Not loving "billing information on file" but I understand it could be a credit card or a contract.

Reply:
Nah this goes away after the new free trial is implemented anyways I think